coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -78,13 +78,13 @@ def parse(self):
             When no parsing method is present for the given language and
             docstyle.
         """
-        if self.language == 'python' and self.docstyle == 'default':
+        if self.language == 'python' and self.docstyle == 'sphinx':
```

Comment:
'default' need to be deprecated properly.

the bears use 'default'.

Ideally, do not change the tests.  if the tests still pass even when you rename the style files, you probably havent broken anything.

Reply:
@jayvdb The tests fail on renaming the default.coalang file to sphinx.coalang

Reply:
Hence this was necessary